stdlib/public/stubs/Stubs.cpp

```diff
@@ -172,13 +172,13 @@ static uint64_t swift_floatingPointToString(char *Buffer, size_t BufferLength,
 extern "C" uint64_t swift_float32ToString(char *Buffer, size_t BufferLength,
                                           float Value) {
   return swift_floatingPointToString<float>(Buffer, BufferLength, Value,
-                                            "%0.*g");
+                                            "%0.9g");
```

Comment:
An easy mistake to make.  Thanks for addressing it.  @dabrahams once that's resolved, I'm fine with this change.